picard/ui/metadatabox.py

```diff
@@ -317,11 +323,18 @@ def contextMenuEvent(self, event):
         menu.exec_(event.globalPos())
         event.accept()
 
+    def add_to_preserved_tags(self, name):
+        preserved_tags = [tag for tag in
+                          map(lambda x: x.strip(), config.setting['preserved_tags'].split(','))
+                          if tag != ""]
+        preserved_tags.append(name)
+        config.setting['preserved_tags'] = ", ".join(list(set(preserved_tags)))
```

Comment:
Just a minor thing i just noticed, we don't need to convert to list, as `str.join()` takes an iterable as argument: `", ".join(set(preserved_tags))`

Ref: https://docs.python.org/2/library/stdtypes.html#str.join